pandas/tests/io/test_parquet.py

```diff
@@ -907,6 +907,26 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    @td.skip_if_no("pyarrow")
+    def test_read_write_attrs(self, pa):
+        pdf = pd.DataFrame({"a": [1]})
```

Comment:
Any reason why to use `pdf` instead of `df`, as in the other tests? If there is no reason, better to be consistent.

Reply:
Sounds good, will do.